ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource

```diff
@@ -92,3 +96,76 @@ Restart Kueue
     IF    ${result.rc} != 0
         FAIL    Failed to finish Kueue rollout
     END
+
+Prepare Training Operator E2E Core Test Suite
+    [Documentation]    Prepare Training Operator E2E Core Test Suite
+    Prepare Training Operator E2E Test Suite    ${KFTO_CORE_BINARY_NAME}
+
+Prepare Training Operator E2E Upgrade Test Suite
+    [Documentation]    Prepare Training Operator E2E Upgrade Test Suite
+    Prepare Training Operator E2E Test Suite    ${KFTO_UPGRADE_BINARY_NAME}
+
+Prepare Training Operator E2E Test Suite
+    [Documentation]    Prepare Training Operator E2E Test Suite
+    [Arguments]        ${test_binary}
+    Log To Console    "Downloading compiled test binary ${test_binary}"
+    ${result} =    Run Process    curl --location --silent --output ${test_binary} ${DISTRIBUTED_WORKLOADS_RELEASE_ASSETS}/${test_binary} && chmod +x ${test_binary}
+    ...    shell=true
+    ...    stderr=STDOUT
+    Log To Console    ${result.stdout}
+    IF    ${result.rc} != 0
+        FAIL    Unable to retrieve ${test_binary} compiled binary
+    END
+    Create Directory    %{WORKSPACE}/codeflare-${test_binary}-logs
+    Enable Component    trainingoperator
+    Wait Component Ready    trainingoperator
+    Log To Console    "Restarting kueue"
+    Restart Kueue
+
+Teardown Training Operator E2E Core Test Suite
+    [Documentation]    Prepare Training Operator E2E Core Test Suite
+    Teardown Training Operator E2E Test Suite    ${KFTO_CORE_BINARY_NAME}
+
+Teardown Training Operator E2E Upgrade Test Suite
+    [Documentation]    Prepare Training Operator E2E Upgrade Test Suite
+    Teardown Training Operator E2E Test Suite    ${KFTO_UPGRADE_BINARY_NAME}
+
+Teardown Training Operator E2E Test Suite
+    [Documentation]    Teardown Training Operator E2E Test Suite
+    [Arguments]        ${test_binary}
+    Log To Console    "Removing test binaries"
+    ${result} =    Run Process    rm -f ${test_binary}
```

Comment:
It would be better to use https://robotframework.org/robotframework/latest/libraries/OperatingSystem.html#Remove%20File